### PR TITLE
Delete descriptions from some sized/graded armor

### PR DESCRIPTION
**Pull request by Uwuewsky**
#### Summary
I18N "Reset descriptions of some sized/graded armor to the item-parent description"

#### Purpose of change
Part of #71112 - P1.
Some of the armor used descriptions for its XL/XS/steel graded variants, while the other half of items did not.

#### Describe the solution
This PR removes variant descriptions where possible, resetting them to the shared description of the base item.

#### Describe alternatives you've considered
Don't merge PR. Sure.

#### Testing
The files are linted, the game loads correctly, the armor in the debug menu shows a general description.

#### Additional context
This reduced the number of translated lines by ~400. This is quite a large amount even for copy-paste.

Fixed a couple of minor bugs along:
XL faux fur winter survivor boots was "copy-from" wrong boots.
Changed a few variants to using "copy-from".
"armguard_hard" did't have "looks_like" (a lot of armor doesnt have this field tho)

### CODE REVIEW

**Comment by GuardianDll** [comment]
Why `looks_like` are also removed?

**Comment by Procyonae** [comment]
> Why `looks_like` are also removed?

Pretty sure looks_like is auto-applied to stuff using copy-from

**Comment by Procyonae** [comment]
I'm reworking gambesons.json if you want to avoid that in P2 or whatever

**Comment by Uwuewsky** [comment]
[Yes, it is written in the docs](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#copy-from-and-abstract)

> `"copy-from"` also implies `"looks_like"` connection, so you don't have to specify the id multiple times.

File: data/json/items/armor/torso_armor.json [hunk 1]
```diff
@@ -625,120 +624,105 @@
...
+    "name": { "str": "mild steel chainmail vest" }
   },
   {
     "id": "xl_lc_chainmail_vest",
     "copy-from": "xl_chainmail_vest",
     "replace_materials": { "steel": "lc_steel_chain" },
     "type": "ARMOR",
-    "name": { "str": "XL mild steel chainmail vest" },
-    "description": "A customized XL mild steel chainmail vest.  It's a sleeveless piece of chainmail with small straps designed to better distribute the weight."
+    "name": { "str": "XL mild steel chainmail vest" }
```
Comment (Maleclypse) [inline data/json/items/armor/torso_armor.json:634]:
```suggestion
    "name": { "str": "XL mild steel chainmail vest" },
```

File: data/json/items/armor/torso_armor.json [hunk 2]
```diff
@@ -625,120 +624,105 @@
...
-    "name": { "str": "mild steel chainmail vest" },
-    "description": "A customized mild steel chainmail vest.  It's a sleeveless piece of chainmail with small straps designed to better distribute the weight."
+    "name": { "str": "mild steel chainmail vest" }
   },
   {
     "id": "xl_lc_chainmail_vest",
     "copy-from": "xl_chainmail_vest",
     "replace_materials": { "steel": "lc_steel_chain" },
     "type": "ARMOR",
     "name": { "str": "XL mild steel chainmail vest" },
```
Comment (Maleclypse) [inline data/json/items/armor/torso_armor.json:634]:
```suggestion
    "name": { "str": "XL mild steel chainmail vest" }
```

**Comment by Maleclypse** [comment]
I made some changes to rebase you to a working test suite.  Though I may have deleted a comma that was necessary on accident and now I can't find it.

**Pull request merged**